sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventParticipantsGrid.java

```diff
@@ -130,14 +131,16 @@ public EventParticipantsGrid(EventParticipantCriteria criteria) {
 
 		getColumn(EventParticipantIndexDto.CONTACT_COUNT).setSortable(false);
 
-		addItemClickListener(new ShowDetailsListener<>(CASE_ID, false, e -> {
-			if (e.getCaseUuid() != null) {
-				ControllerProvider.getCaseController().navigateToCase(e.getCaseUuid());
-			} else if (e.getInJurisdiction()) {
-				EventParticipantDto eventParticipant = FacadeProvider.getEventParticipantFacade().getEventParticipantByUuid(e.getUuid());
-				ControllerProvider.getCaseController().createFromEventParticipant(eventParticipant);
-			}
-		}));
+		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.CASE_SURVEILANCE)) {
```

Comment:
you should check also the CASE_VIEW right, please use `UiUtil.isPermitted`

Reply:
changed